arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts

```diff
@@ -255,6 +255,142 @@
 		regulator-max-microvolt = <3700000>;
 	};
 
+	vreg_cam_vio_1p8: cam-vddio {
+		compatible = "regulator-fixed";
+		regulator-name = "vreg_cam_vio_1p8";
+
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
+
+		regulator-enable-ramp-delay = <10>;
+
+		gpio = <&tlmm 49 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		regulator-always-on;
```

Comment:
why always-on?

Reply:
Actually it works without always-on, but cci spams a lot of timeouts and i2cdetect works with delays

Reply:
> Actually it works without always-on, but cci spams a lot of timeouts and i2cdetect works with delays

Since there is no obvious solution, I mark it as resolved